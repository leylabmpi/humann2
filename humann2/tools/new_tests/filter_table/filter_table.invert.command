# basic prevalence filter
humann2_filter_table \
	     -i filter_table.input \
	     -L M2 \
	     -o filter_table.invert.output \
	     -p 75 \
	     --invert \