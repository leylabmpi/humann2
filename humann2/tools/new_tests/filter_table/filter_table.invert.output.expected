# SAMPLES	S1	S2	S3	S4
M1	-	-	-	-
M2	-	-	-	-
F2	2	0	0	0
F2|S1	1	0	0	0
F2|S2	1	0	0	0
